hieradata/production.yaml

```diff
@@ -53,6 +53,7 @@ govuk::apps::publisher::email_group_business: 'publisher-alerts-business@digital
 govuk::apps::publisher::email_group_citizen: 'publisher-alerts-citizen@digital.cabinet-office.gov.uk'
 govuk::apps::publisher::email_group_force_publish_alerts: 'mainstream-force-publishing-alerts@digital.cabinet-office.gov.uk'
 govuk::apps::publisher::govuk_notify_template_id: "e00e89f5-b622-4dcb-8f30-e6c70231a940"
+govuk::apps::publisher::fact_check_subject_prefix: ''
```

Comment:
I wasn't sure what would be best here. In the code, if it's unset, it'll default to dev, so that it doesn't need an environment variable set when running locally etc. But an empty string is also a weird value to have.

Maybe setting it to 'production' but handling that value specially? But that also seems weird.

Reply:
Hmm I see your point, could we not update the application code to default to an empty string if no env variable is set which would mitigate the need to set it here? You would also have a safety net if for whatever reason the variable wasn't set on the machine in prod. 

The [email groups config](https://github.com/alphagov/publisher/blob/master/config/initializers/email_groups.rb) also follows this pattern of setting the prod variables as default if nothing is set, what do you think?

Reply:
Perhaps I'm overthinking this: I didn't want the production behaviour to be default because I didn't want a misconfigured non-production machine to send emails that looked like production. But it would certainly be simpler to have the production value as the default. (And thinking about it, dev machines won't be able to send emails anyway.)

Reply:
Yup and as you say - unless the address is whitelisted it wouldn't receive an email anyway if some non prod machine was borked. Sorry this comment involves work across both PR's!